src/taipy/rest/api/resources/cycle.py

```diff
@@ -33,13 +32,80 @@ class CycleResource(Resource):
     get:
       tags:
         - api
-      summary: Get a cycle
-      description: Get a single cycle by ID
+      description: |
+        Returns a `CycleSchema^` representing the unique `Cycle^` identified by the `cycle_id`
```

Comment:
\`cycle_id\` -> \*cycle_id\*

(all parameter names should be in italics, not fixed-size, which is reserved for the API elements)

(True everywhere)